codec/x264/encoder_test.go

```diff
@@ -0,0 +1,98 @@
+package x264
+
+import (
+	"bytes"
+	"context"
+	"github.com/edaniels/golog"
+	"github.com/nfnt/resize"
+	"go.viam.com/test"
+	"image"
+	"image/color"
+	"image/jpeg"
+	"image/png"
+	"os"
+	"testing"
+)
+
+var width int
+var height int
+var logger golog.Logger
+var imgCyan image.Image
+var imgFuchsia image.Image
+var w bool
+
+const DefaultKeyFrameInterval = 30
+
+func init() {
```

Comment:
[minor] instead of having globals, just have a common benchmark setup function that makes these for you. then before the benchmark, call b.Reset to exclude all work done for these.